web/pgadmin/browser/server_groups/servers/databases/schemas/synonyms/tests/utils.py

```
@@ -70,7 +70,7 @@ def verify_synonym(server, db_name, synonym_name):
                                              server['host'],
                                              server['port'])
         pg_cursor = connection.cursor()
-        pg_cursor.execute("SELECT * FROM pg_synonym WHERE synname='%s'" %
+        pg_cursor.execute("SELECT * FROM pg_catalog.pg_synonym WHERE synname='%s'" %
                           synonym_name)
```

Ruff (0.15.15)

[error] Possible SQL injection vector through string-based query construction

(S608)